src/app/features/android/android-interface.ts

```diff
@@ -56,6 +56,28 @@ export interface AndroidInterface {
     progress: number, // -1 => undefined; 999 => indeterminate; 333 => show play but no progress bar
   ): void;
 
+  // WebDAV
+  makeHttpRequestWrapped(
+    url: string,
+    method: string,
+    data: string,
+    username: string,
+    password: string,
+    readResponse: boolean,
+  ): Promise<object>;
+
+  makeHttpRequest?(
+    rId: string,
+    url: string,
+    method: string,
+    data: string,
+    username: string,
+    password: string,
+    readResponse: boolean,
+  ): void;
+
+  makeHttpRequestCallback(rId: string, result: Object): void;
```

Comment:
The linting doesn't like object. If you're sure that this is an object you can use something like 
```
{[key: string]: string | number}
``` 
Or possibly `unknown` or even `any`

Reply:
Will `{}` work? I haven't used TS before 😅

Reply:
No unfortunately not :) The quickest compromise might be:
```
{[key: string]:  any }
```